app/src/main/java/org/mozilla/tv/firefox/utils/ServiceLocator.kt

```diff
@@ -76,7 +77,13 @@ open class ServiceLocator(val app: Application) {
     val engineViewCache by lazy { EngineViewCache(sessionRepo) }
     val sessionManager get() = app.webRenderComponents.sessionManager
     val sessionUseCases get() = app.webRenderComponents.sessionUseCases
-    val searchEngineManager = SearchEngineManager().apply { GlobalScope.launch { load(app) } }
+    val searchEngineManager by lazy {
+        val replacements = mapOf("google" to "google-b-amzftv", "google-2018" to "google-b-1-amzftv",
+            "google-b-m" to "google-b-amzftv", "google-b-1-m" to "google-b-1-amzftv")
+        val engineProvider = SearchEngineProviderWrapper(replacements)
+
+        SearchEngineManager(listOf(engineProvider)).apply { GlobalScope.launch { load(app) } }
```

Comment:
Nit: the `ServiceLocator` is already pretty noisy, do you think we could move this into a simple factory to cut down on how much we add to that?

Reply:
I'm not sure how to do that, would need some help.

Reply:
Took care of it in the above mentioned PR: https://github.com/mkaply/firefox-tv/pull/1